doc/examples/reconst_qtiplus.py

```diff
@@ -0,0 +1,263 @@
+performing short data acquisition which can correspond to scanning patients
+in clinical settings.
+
+The full dataset used in this tutorial was originally published at
+https://github.com/filip-szczepankiewicz/Szczepankiewicz_DIB_2019,
+and is described in [3]_.
+
+"""
+
+"""
+First, let's load the complete dataset and create the gradient table.
+We mark these two with the '_217' suffix.
+
+"""
+
+data_img, mask_img, gtab_217 = read_DiB_217_lte_pte_ste()
+data_217 = data_img.get_fdata()
+mask = mask_img.get_fdata()
+
+"""
+Second, let's load the downsampled dataset and create the gradient table.
+We mark these two with the '_70' suffix.
+
+"""
+
+data_img, _, gtab_70 = read_DiB_70_lte_pte_ste()
+data_70 = data_img.get_fdata()
+
+"""
+Now we can fit the QTI model to the datasets containing 217 measurements, and
+use it as reference to compare the constrained and unconstrained fit on the
+smaller dataset. For time considerations, we restrict the fit to a
+single slice.
+"""
+
+mask[:, :, :13] = 0
+mask[:, :, 14:] = 0
+
+qtimodel_217 = qti.QtiModel(gtab_217)
+qtifit_217 = qtimodel_217.fit(data_217, mask)
+
+
+"""
+Now we can fit the QTI model using the default unconstrained fitting method
+to the subsampled dataset:
+"""
+
+qtimodel_unconstrained = qti.QtiModel(gtab_70)
+qtifit_unconstrained = qtimodel_unconstrained.fit(data_70, mask)
+
+"""
+Now we repeat the fit but with the constraints applied.
+To perform the constrained fit, we select the 'SDPdc' fit method when creating
+the QtiModel object.
+Note: this fit method is slower compared to the defaults unconstrained.
+
+If mosek is installed, it can be specified as the solver to be used
+as follows:
+
+qtimodel = qti.QtiModel(gtab, fit_method='SDPdc', cvxpy_solver='MOSEK')
+qtifit = qtimodel.fit(data, mask)
+
+If Mosek is not installed, the constrained fit can still be performed, and
+SCS will be used as solver. SCS is typically much slower than Mosek, but
+provides similar results in terms of accuracy. To give an example, the fit
+performed in the next line will take approximately 15 minutes when using SCS,
+and 2 minute when using Mosek!
+"""
+
+qtimodel_constrained = qti.QtiModel(gtab_70, fit_method='SDPdc')
+qtifit_constrained = qtimodel_constrained.fit(data_70, mask)
+
+"""
+Now we can visualize the results obtained with the constrained and
+unconstrained fit on the small dataset, and compare them with the
+"ground truth" provided by fitting the QTI model to the full dataset.
+For example, we can look at the FA and $\mu$FA maps, and their value
+distribution in White Matter in comparison to the ground truth.
+"""
+
+from dipy.viz.plotting import compare_qti_maps
+
+z = 13
+wm_mask = qtifit_217.ufa[:, :, z] > 0.6
+compare_qti_maps(qtifit_217, qtifit_unconstrained, qtifit_constrained, wm_mask)
+
+"""
+The results clearly shows how many of the FA and $\mu$FA values
+obtained with the unconstrained fit falls outside the correct
```

Comment:
As you can see below, syntax error:

<img width="792" alt="image" src="https://user-images.githubusercontent.com/23106443/166962626-f6463079-da6c-4182-b983-e20bd7d536dd.png">

I suppose, it needs a space